src/main/scala/beam/analysis/cartraveltime/CarTripStatsFromPathTraversalEventHandler.scala

```diff
@@ -159,10 +164,19 @@ class CarTripStatsFromPathTraversalEventHandler(
       getIterationCarRideStats(event.getIteration, singleRideStats)
     }
 
+    iterationsCarTripInfo ++= type2Statistics.map {
+      case (carType, iterationCarTripStats) => (event.getIteration, carType) -> iterationCarTripStats
+    }
+
     averageCarSpeedPerIterationByType += type2Statistics.mapValues(_.speed.stats.avg)
 
+    //FIXME: create chart and store the image every time iteration ends... Why so?
```

Comment:
This is expected behavior, we re-created the graph in the end of every iteration. Over the iterations the graph contains more data point